tutorials/getting_started/configuration.md

```diff
@@ -0,0 +1,344 @@
+---
+layout: tutorial
+title: A Deeper Dive Into Configuring Experiments
+id: configuring-experiments
+---
+
+Now that we know how to train and evaluate models,
+let's take a deeper look at our experiment configuration file,
+[tutorials/getting_started/simple_tagger.json](https://github.com/allenai/allennlp/blob/master/tutorials/getting_started/simple_tagger.json).
+
+The configuration is a JSON (or [HOCON](https://github.com/typesafehub/config/blob/master/HOCON.md)) object
```

Comment:
I find this confusing.  Is it JSON or HOCON?  Perhaps you mean "The configuration is HOCON (but don't worry if you are not familiar with HOCON--JSON is a valid subset of HOCON)"